### PR TITLE
Add monitoring alerts

### PR DESCRIPTION
**Pull request by jacobwindsor**
- Adds alerts to the monitoring dashboard
  - If a pod fails
  - If ingest-core memory usage is coming close to the maximum
  - If we are coming close to the number of pods allowed on a node (70), the threshold for alert set at 65
  - If the replica count of ingest-exporter exceeds 10
- Adds a slack notification channel to report alerts to #hca-ingest-dev-notifications
- Makes it easier to deploy changes to the monitoring dashboard

### CODE REVIEW

**Comment by jacobwindsor** [comment]
FYI these changes are already deployed to the monitoring environments. I did that for testing purposes.

**Review by ke4 — APPROVED** [review]
Looks fine with me 👍

**Review by amnonkhen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -121,9 +121,15 @@ Coming soon
   - Login with `admin` and the result from step 4
 
 ### Upgrading Ingest Monitoring Dashboard
```
Comment (amnonkhen) [inline README.md:123]: Why is the **monitoring** section under the U**pgrade Backend Services** section?

**Review by amnonkhen — CHANGES_REQUESTED** [review]
The readme files seems to be getting very large. I would split.

**Comment by jacobwindsor** [comment]
@amnonkhen split how? Into multiple READMEs?

**Review by amnonkhen — APPROVED** [review]

**Pull request merged**